src/atomicui/pages/DemoPlaceholderPage/DemoPlaceholderPage.tsx

```diff
@@ -108,7 +108,7 @@ const DemoPlaceholderPage: React.FC<DemoPlaceholderPageProps> = ({
 					onCloseSidebar={() => {}}
 					onOpenSignInModal={() => {}}
 					isGrabVisible={isGrabVisible}
-					showGrabDisclaimerModal={show.grabDisclaimerModal}
+					showGrabDisclaimerModal={false}
```

Comment:
This is the placehoder page, which is shown to user when credentials are null. This is used to provide user a non blocking loading experience in case when their credentials are flushed, or when map styles or provider is changed. @vinguthh